checker/src/main/java/org/checkerframework/checker/index/qual/EnsuresLTLengthOf.java

```diff
@@ -78,4 +80,20 @@
     @JavaExpression
     @QualifierArgument("offset")
     String[] offset() default {};
+
+    /** An inner annotation which makes {@link EnsuresLTLengthOf} annotation repeatable */
```

Comment:
Small comments:
1. End each sentence with punctuation, such as a period.
2. Use "that" instead of "which" in this context.  Here is an explanation from _Strunk and White_:
> That.  Which.  _That_ is the defining, or restrictive pronoun, _which_ the
> nondefining, or nonrestrictive.
>
> 	The lawn mower that is broken is in the garage.  (Tells which one.)
> 	The lawn mover, which is broken, is in the garage.  (Adds a fact
> 		about the only mower in question.)
>
> The use of _which_ for _that_ is common in written and spoken language
> ("Let us now go even unto Bethlehem, and see this thing which is come to
> pass.")  Occasionally _which_ seems preferable to _that_, as in the
> sentence from the Bible.  But it would be a convenience to all if these two
> pronouns were used with precision.  The careful writer, watchful for small
> conveniences, goes _which_-hunting, removes the defining _whiches_, and by
> so doing improves his work.
>
> The Elements of Style
> William Strunk, Jr. and E. B. White

Reply:
Fixed that!